pkg/monitoring/domainstats/prometheus/prometheus.go

```diff
@@ -243,35 +243,39 @@ func (metrics *vmiMetrics) updateCPUAffinity(cpuMap [][]bool) {
 	)
 }
 
+func nanosecondsToSeconds(ns uint64) float64 {
+	return float64(ns) / 1000000000
+}
+
```

Comment:
@machadovilaca There is a change in the behavior here. Previously, the division was executed between two integers and then the outcome (an `int`) was cast to `float64`. Rigth now a division from a `float64` and a `int`, and the outcome is a `float64`. This causes that previously the outcome was rounded, while right now it is not anymore. This is the cause of the failing migration tests, since in https://github.com/kubevirt/kubevirt/blob/2922b84415bbd5fd884bd004903212907bbd328c/tests/monitoring/prometheus_utils.go#L102 we try to convert the result in a `int`.
I am quite conviced that this implementation is better, but in this case we need to adjust https://github.com/kubevirt/kubevirt/blob/2922b84415bbd5fd884bd004903212907bbd328c/tests/monitoring/prometheus_utils.go#L102, managing `float` and not `int`.

Reply:
updated

Reply:
@sradco Can you confirm that this change is ok? Thanks